printer_mods/JosAr/Klicky-Probe/README.md

```diff
@@ -0,0 +1,118 @@
+# Klicky-Probe
+Microswitch probe with magnetic attachement, primarily aimed at CoreXY 3d printers with a focus on the Voron printers, should work on other printers with the variable mount.
+
+The objectives for this project are:
+- drop in replacement for Omron TL-Q5MC2 or PL-08N2 (you don't need to replace the toolhead)
+- easier and faster to build that similiar probe types
```

Comment:
Replace `that` with `than`.

Replace `similiar` with `similar`.

Reply:
Please also change `faster to built that` to `faster to build than`.

Reply:
Sorry for that, slipped past, corrected.